go/externals/implicit_team_test.go

```diff
@@ -146,20 +143,12 @@ func TestPartImplicitTeamTLFNameEvenMore(t *testing.T) {
 		{"/keybase/private/#alice", nil},
 	}
 
-	deepEq := func(a, b keybase1.ImplicitTeamDisplayName) bool {
-		x, _ := libkb.MsgpackEncode(a)
-		y, _ := libkb.MsgpackEncode(b)
-		fmt.Printf("%s\n", hex.EncodeToString(x))
-		fmt.Printf("%s\n", hex.EncodeToString(y))
-		return bytes.Equal(x, y)
-	}
-
 	for _, test := range tests {
-		itn, err := libkb.ParseImplicitTeamTLFName(MakeAssertionContext(), test.input)
+		itn, err := kbname.ParseImplicitTeamTLFName(MakeAssertionContext(), test.input)
 		if test.output == nil {
 			require.Error(t, err)
 		} else {
-			require.True(t, deepEq(itn, *test.output))
+			require.Equal(t, *test.output, itn)
```

Comment:
require already does deep equality and provides a better comparison view on failure